### ISSUE TITLE
When you use Open, Save, Export etc Windows PC makes a weird like hdd crashing sound

### ISSUE THREAD

**Issue by athan01** [xournalpp/xournalpp#2365]
- OS: Windows
 - Which version of libgtk do you use: 3.24.23
 - Version of Xournal++: 1.0.19, d376e7a6, specific branch ?
 - Installation method: official release

When you use Open, Save, Export etc Windows PC makes a weird like hdd crashing sound. I checked out xournal too and it does the same thing.

Just try to open, save or export any file to a windows 10 pc

I expected that I wouldn't hear such a sound.

**Comment by LittleHuba** [comment]
Could you go into more detail what this sound is? Is it generated by software or actually by your hardware? Can you give a recording of this sound?

**Comment by Technius** [comment]
Does this happen in general or only with Xournal(++)?

This could be a sign of impending hard drive failure. You should start backing up your important files and maybe use software like CrystalDiskInfo to confirm the status of your hard drive.

**Comment by athan01** [comment]
No, it happens only with xournal(++) but I'll certainly do this just to be safe.Στις 4 Νοε 2020 3:12 π.μ., ο χρήστης Bryan Tan <notifications@github.com> έγραψε:
Does this happen in general or only with Xournal(++)?
This could be a sign of impending hard drive failure. You should start backing up your important files and maybe use software like CrystalDiskInfo to confirm the status of your hard drive.
